data/commodities.txt

```diff
@@ -430,6 +528,7 @@ trade
 		"capacitors"
 		"circuit boards"
 		"cloud chambers"
+		"conic cegrameters"
```

Comment:
```suggestion
```
I'm not sure what this is supposed to be, but I couldn't find anything similar.

Reply:
It's just something I made up. In the 31st century there is no doubt a lot of new technology that we do not currently have, so it wouldn't make sense to have only items from the 21st century. My philosophy is that ideally there should be a mix of current and futuristic technologies to add plausibility that the game takes place in the future while also providing familiarity for the player. A couple of made up items are already in the file, such as a couple of creative names in the illegal substances category. I decided to try my hand at creating some in other categories. If people don't like them it is alright with me if they aren't included, but I would like to hear people's opinions.

Reply:
I personally like having stuff that doesn't exist (yet). It adds to the feel that ES society is substantially different from today's society, which I don't feel like there's much of. It just feels like our current civilization but in space. Transporting tech that's unique to the 31st century helps mitigate that a bit.